pkg/replication/replicator.go

```diff
@@ -108,20 +108,22 @@ func (r replicator) InitializeReplication(
 		ignoreControllers,
 		concurrentRealityRequests,
 		true,
+		false,
 	)
 }
 
 func (r replicator) InitializeReplicationWithCheck(
```

Comment:
Just `InitializeDaemonSetReplication(concurrentRealityRequests int)` would maybe make a lot more sense than these arguments. Why does it have to be the case that you can't pass in `overrideLock`?